server/src-lib/Hasura/RQL/DDL/Schema/Cache.hs

```diff
@@ -276,6 +277,22 @@ buildSchemaCacheRule = proc (catalogMetadata, invalidationKeys) -> do
              |)
         >-> (\actionMap -> returnA -< M.catMaybes actionMap)
 
+      -- scheduled triggers
+      scheduledTriggersMap <- (mapFromL _cstName scheduledTriggers >- returnA)
+        >-> (| Inc.keyed (\_ (CatalogScheduledTrigger n wc s p rc h ) -> do
```

Comment:
Could you use more meaningful names here? Maybe use record selectors or `RecordWildCards`?